style.css

```diff
@@ -49,7 +49,7 @@ p {
   justify-content: flex-start;
 }
 
-.socials {
+.social {
   list-style: none;
   color: rgb(20, 126, 126);
```

Comment:
- [ ] Kindly adjust the font size, font family, and font color of your design to match that of the Figma template as much as possible. you can get the font details from the Figma template.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/185556965-6baad949-b648-408f-ad43-6c6f3a48ccd8.PNG)
![Capture1](https://user-images.githubusercontent.com/85095702/185556988-781fb340-0706-444b-a0b5-9bd4707e3edf.PNG)
Figma template
![figma](https://user-images.githubusercontent.com/85095702/185557048-f1862cfd-360f-4206-ab39-a197f89e8502.PNG)
![figma1](https://user-images.githubusercontent.com/85095702/185557064-2cd6ddc5-81d4-4f1b-ab6e-9c5257d144d8.PNG)

- [ ] Also, kindly add the background decorations from the design template to your design